payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherConfirmationActivity.kt

```diff
@@ -5,17 +5,18 @@ import android.content.Intent
 import android.os.Bundle
 import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import com.stripe.android.view.AuthActivityStarterHost
+import kotlinx.coroutines.launch
 
 /**
- * WIP - host activity to perform actions for PaymentLauncher.
+ * Host activity to perform actions for [PaymentLauncher].
  * This activity starts activities to handle next actions, capture their result
  * and convert them to [PaymentResult] and return back to client.
  */
 internal class PaymentLauncherConfirmationActivity : AppCompatActivity() {
 
-    private val viewModel: PaymentLauncherViewModel by viewModels {
-        PaymentLauncherViewModel.Factory()
-    }
+    private lateinit var launcherArgs: PaymentLauncherContract.Args
```

Comment:
Is this necessary?

Reply:
Yes, in the follow up implementation, this field is needed by test to mock the `viewModelFactory`, like what [PaymentOptionsActivity](https://github.com/stripe/stripe-android/blob/a9bdc145ee5ec64f94a7406fff396f0eb847a944/payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt#L39) does.